CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
 ## Unreleased
 
+### Fixed
+
+- Fixed Nullpointer in `Manifestation.toString()`
```

Comment:
Der Fehler war aber doch in `dumpShortenedRelations`, oder?

Reply:
ja, aber das wurde aus `toString()` heraus aufgerufen